src/librustdoc/html/static/themes/ayu.css

```diff
@@ -0,0 +1,474 @@
+/*
```

Comment:
Your CSS isn't up-to-date: missing the scrollbar rules. Which makes me wonder if the "default" themes are tested...